### DISCUSSION TITLE
Does DBeaver supports Oracle GDS connections?

### DISCUSSION THREAD

**Discussion by LmG7119** [dbeaver/dbeaver#20711] [DBeaver Development]
Does DBeaver supports Oracle GDS connections using TNS names?

**Comment by LonwoLonwo** [comment]
Hello @LmG7119

I'm not sure on the 100% about GDS, but you can try to connect to Oracle using TNS names:

![2023-07-27 10_37_05-Connect to a database](https://github.com/dbeaver/dbeaver/assets/45152336/5f10c1bc-051b-4e89-bd48-c97d2f8faaec)

**Comment by LmG7119** [comment]
That is a problem. I can connect just fine to Oracle using TNSname that are not set up with GDS but not with TNSname with GDS. I don't know if there is an additional option for jdbc to specify GDS or not.
